README.md

```diff
@@ -16,7 +16,7 @@ governing permissions and limitations under the License.
 # Template Registry API
 
 ## OpenAPI Schema
-Template Registry API follows OpenAPI 3.0.3 Specification.
+Template Registry API follows OpenAPI 2.0 Specification.
```

Comment:
Are we switching back to  OpenAPI 2.0 version?

Reply:
unfortunately yes, we have to because runtime doesn't support openapi 3.0 for the scope and client validation features yet ☹️

Reply:
this pr https://github.com/adobe/adp-template-registry-api/pull/37 reverts us back to 2.0

Reply:
Ahh damn, gotcha